services/core-web/src/components/Forms/GenerateDocumentForm.js

```diff
@@ -38,36 +53,70 @@ const createFields = (fields) => (
   </div>
 );
 
-export const GenerateDocumentForm = (props) => (
-  <Form layout="vertical" onSubmit={props.handleSubmit}>
-    <Row gutter={16}>
-      <Col span={24}>{createFields(props.documentType.document_template.form_spec)}</Col>
-    </Row>
-    <div className="right center-mobile">
-      <Popconfirm
-        placement="topRight"
-        title="Are you sure you want to cancel?"
-        onConfirm={props.closeModal}
-        okText="Yes"
-        cancelText="No"
-        disabled={props.submitting}
-      >
-        <Button className="full-mobile" type="secondary" disabled={props.submitting}>
-          Cancel
+export const GenerateDocumentForm = (props) => {
+  const [fileType, setFileType] = useState("PDF");
+
+  return (
+    <Form layout="vertical" onSubmit={props.handleSubmit}>
+      {props.allowDocx && (
+        <div>
+          <Row key="fileType">
+            <Col span={24}>
+              <>
+                <Form.Item>
+                  <Field
+                    id="file_type"
+                    name="file_type"
+                    label="Select the file type for this document"
+                    data={fileTypes}
+                    component={RenderSelect}
+                    onChange={(value) => setFileType(value)}
+                  />
+                </Form.Item>
+                {fileType === "DOCX" && (
+                  <>
+                    <Alert
+                      description="If you plan to edit this document, please ensure that you do not change any vital information that would contradict BC government legislation pertaining to privacy, or financial information."
```

Comment:
"privacy, or financial information" --> "privacy or financial information"?

Reply:
copied from the ticket, but I agree the comma is not needed